pom.xml

```diff
@@ -22,13 +24,53 @@
     <tag>HEAD</tag>
   </scm>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.1</version>
```

Comment:
version already specified by the parent pom, I think we shouldn't specify it (or have a comment why it's important to differ/stay on a given version).

Reply:
Yes, will remove